scripts/lint.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if ! ./bin/prettier.js --list-different index.js src/*.js bin/*.js; then
```

Comment:
it's annoying that we have to list those paths in two different places. Do you think you could come up with a way to share the paths?